R/theme-elements.r

```diff
@@ -480,6 +480,7 @@ el_def <- function(class = NULL, inherit = NULL, description = NULL) {
   strip.placement.y   = el_def("character", "strip.placement"),
   strip.switch.pad.grid = el_def("unit"),
   strip.switch.pad.wrap = el_def("unit"),
+  default.scales      = el_def("list"),
```

Comment:
It's probably best to add a specific element class here. PR #2749 adds `element_geom()`, so maybe this one here could be called `element_scales()`. But I'd like to have input from @yutannihilation or @thomasp85 here.

Reply:
Also, if we add a specific element class then we check then and there that the element values are functions and then we can avoid the `mode()` check in `find_global()` (to be moved into `find_scale()`).